### PR TITLE
Feature/create post

### PR DESCRIPTION
**Pull request by Nutchayaporn**
Create post feature

### CODE REVIEW

**Review by aomsk — APPROVED** [review]

File: backend/databases/database.go [hunk 1]
```diff
@@ -36,8 +35,8 @@ func Connect_DB() *sql.DB {
 	//Create psql Info for connect your Postgres DB
 	psqlInfo := fmt.Sprintf("host=%s port=%d user=%s "+
 		"password=%s dbname=%s sslmode=disable",
-		// host, port, "postgres", "1234", "Market")
-		host, port, os.Getenv("POSTGRES_USER"), os.Getenv("POSTGRES_PASSWORD"), os.Getenv("POSTGRES_DB"))
```
Comment (SilverSky9) [inline backend/databases/database.go:40]: Pls, use env file for connect DB

File: frontend/pages/Home.js [hunk 2]
```diff
@@ -119,7 +147,7 @@ const Matching = ({ tag }) => {
         async function getPost() {
             let response = await fetch("http://localhost:3000/post/geybymultitag/" + url + ',')
```
Comment (SilverSky9) [inline frontend/pages/Home.js:148]: Pls, Use Backend IP for use query

File: frontend/pages/Home.js [hunk 3]
```diff
@@ -192,8 +373,34 @@ const Matching = ({ tag }) => {
...
+
         </div>
+
+
     )
 }
 
+Matching.getInitialProps = async (ctx) => {
+    // const router = useRouter()
+    const tag = await fetch("http://localhost:3000/tag/getall")
```
Comment (SilverSky9) [inline frontend/pages/Home.js:390]: *

File: frontend/pages/index.js [hunk 4]
```diff
@@ -155,10 +124,13 @@ const Matching = ({ allTag = [] }) => {
 
 
     )
+}
+Matching.getInitialProps = async (ctx) => {
+    const res = await fetch('http://localhost:3000/tag/getall')
```
Comment (SilverSky9) [inline frontend/pages/index.js:129]: *

File: frontend/pages/Home.js [hunk 5]
```diff
@@ -140,6 +168,30 @@ const Matching = ({ tag }) => {
...
+            location: location,
+            phone_number: phone,
+            price: parseInt(price),
+            amount: parseInt(amount),
+            tag_name: category
+            // product_name: productName,
+            // product_option: radioValue,
+            // price: price 
+        }
+        await axios.post('http://localhost:3000/post/create', data)
```
Comment (SilverSky9) [inline frontend/pages/Home.js:184]: *

**Review by SilverSky9 — CHANGES_REQUESTED** [review]
Pls, Change to do

**Pull request closed without merging**